book.bib

```diff
@@ -519,3 +519,19 @@ @article{Chao1984
     url = {https://www.jstor.org/stable/4615964},
     year = {1984}
 }
+
+@article{LahtiL2014,
+    author = {Lahti L et al.},
+    title = {Tipping elements in the human intestinal ecosystem},
+    journal = {Nature Communications},
+    doi = {https://doi.org/10.1038/ncomms5344},
+    year = {2014}
+}
+
+@article{SaloS2021,
```

Comment:
Use the full author surname for references "Salosensaari2021" so we are consistent. Makes managing easier,